pyproject.toml

```diff
@@ -150,3 +150,6 @@ line-length = 100
 # The tool looks to report 'imported but unused (F401)' while actually the function is used by
 # package which import the __init__.py
 "**/__init__.py" = ["F401"]
+
+[tool.setuptools_scm]
```

Comment:
not sure what this is !

Reply:
maybe worth a small comment then !

Reply:
https://github.com/pypa/setuptools_scm/issues/953

What would you add? A link to the issue or just a message saying that this is to avoid warnings?

Reply:
no yeah I just meant like a `# Tool that enables avoiding warnings` or something like, not very fancy, just to make things easier to understand !